.github/workflows/merge-schedule.yaml

```
@@ -0,0 +1,24 @@
+name: Merge Scheduler
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+  schedule:
+    # https://crontab.guru/every-hour
+    - cron: 0 6 * * *
```

That's not "at every hour". This is "at 6am" (every day).

---

I'd just remove that comment to avoid confusion
```suggestion
    - cron: 0 6 * * *
```

---

Yes, that's correct. We don't need to job to run every hour.